src/monitor/db/model.py

```diff
@@ -0,0 +1,250 @@
+from peewee import *
+import datetime
+from src.common.TCP_messages import *
+import json
+import os.path
+from playhouse.shortcuts import model_to_dict
+
+filename = ''
+with open('db_config.json') as config_file:
+    config = json.load(config_file)
+    filename = config['path']
+
+if filename is '':
+    raise Exception("Config file: \"db_config.json\" with field \"path\" is required.")
+
+db = SqliteDatabase(filename)
+
+
+class BaseModel(Model):
+    class Meta:
+        database = db
+
+
+class Sensor(BaseModel):
+    identifier = IntegerField()
```

Comment:
identifier jest adresem MAC sensora, gdyż tylko to jesteśmy pewni, że jest unikalne. W związku z tym muszą być dwa pola:
* id - integer (autoincrement z sekwencją)
* identifier - char field (adres mac)

I najpierw musimy sprawdzić czy dany sensor już mamy w bazie:
* jeśli tak, to tylko aktualizujemy metadane i pomiary i zwracamy id sensora i pomiarów
*jeśli nie, to rejestrujemy wszystkie nowe dane i zwracamy id sensora i pomiarów